src/components/TradeWidget/index.tsx

```diff
@@ -321,13 +322,27 @@ function _getReceiveTokenTooltipText(sellValue: string, receiveValue: string): s
   }
 }
 
+function calculateReceiveAmount(priceValue: string, sellValue: string): string {
+  let receiveAmount = ''
+  if (priceValue && sellValue) {
+    const sellAmount = parseBigNumber(sellValue)
+    const price = parseBigNumber(priceValue)
+
+    if (sellAmount && price) {
+      receiveAmount = sellAmount.multipliedBy(price).toString(10)
+    }
+  }
+
+  return receiveAmount
+}
+
 const TradeWidget: React.FC = () => {
   const { networkId, isConnected, userAddress } = useWalletConnection()
   const { connectWallet } = useConnectWallet()
   const [, dispatch] = useGlobalState()
 
   // Avoid displaying an empty list of tokens when the wallet is not connected
-  const fallBackNetworkId = networkId ? networkId : Network.Mainnet // fallback to mainnet
+  const fallBackNetworkId = networkId ?? Network.Mainnet // fallback to mainnet
```

Comment:
Not sure if `??` is a good idea
Before it was essentially `networkId || Network.Mainne`. For `networkId = 0` which we do have very rarely as a fallback in `WalletApi._getAsyncWalletInfo`, it would default to mainnet. Now no more.
Maybe ok, maybe not ok.

Reply:
Yeah, thought about that too. Will change to `||` then.